.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: go
 osx_image: xcode9.2
+go_import_path: github.com/confluentinc/confluent-kafka-go
```

Comment:
Why is this needed?

Reply:
The code in `examples/` uses the github.com import path. We're using https://docs.travis-ci.com/user/languages/go/#go-import-path to make sure that the Go compiler uses the code from this commit, instead of going to github and grabbing the latest master because the path doesn't exist.